### PR TITLE
Typographical fixes

### PR DESCRIPTION
**Pull request by jurf**
Note: these are purely typographical fixes, nothing else changed.

However, I think that the punchline in the title is wrongly capitalised, but I’m not sure. Also, the 404 doesn’t work.

### CODE REVIEW

File: source/about.html.haml [hunk 1]
```diff
@@ -41,4 +41,4 @@ description: About Flatpak
         %p
           Issues with this website can be reported through its
           = succeed "." do
-            =link_to "Github project", "https://github.com/flatpak/flatpak.github.io/issues"#
+            =link_to "GitHub project", "https://github.com/flatpak/flatpak.github.io/issues"#
```
Comment (AsciiWolf) [inline source/about.html.haml]: Please, fix `.right.toneddown Github project` as well.

Comment (jurf) [reply source/about.html.haml]: Nice catch!

**Comment by jimmac** [comment]
Is it necessary to drop spaces when using em dash? Looks a little awkward to me.

**Comment by jurf** [comment]
Well, you could leave them, but it’s just a matter getting used to it TBH. I really like the look, even though I prefer British style.

Some people space it, some thin-space it, but usually they don’t – Wikipedia [doesn’t](https://en.wikipedia.org/wiki/Dash) in its article about dashes, and neither does [Practical Typography](https://practicaltypography.com/hyphens-and-dashes.html).

**Comment by jurf** [comment]
But I can add the spaces if you want, even though that looks _way_ more awkward in my opinion.

**Comment by jurf** [comment]
Thanks!

**Pull request merged**